hudi-hadoop-common/src/main/java/org/apache/hudi/storage/hadoop/HoodieHadoopStorage.java

```diff
@@ -213,6 +267,16 @@ public Configuration unwrapConf() {
     return fs.getConf();
   }
 
+  @Override
+  public StorageConfiguration<?> buildInlineConf(StorageConfiguration<?> storageConf) {
+    return HadoopInLineFSUtils.buildInlineConf(storageConf);
+  }
+
+  @Override
+  public StoragePath getInlineFilePath(StoragePath outerPath, String origScheme, long inLineStartOffset, long inLineLength) {
+    return HadoopInLineFSUtils.getInlineFilePath(outerPath, origScheme, inLineStartOffset, inLineLength);
+  }
```

Comment:
I think this is not the logic that `HoodieStorage` implementation should care about.  Inline Path is invariant compared to different `HoodieStorage` implementations.

Reply:
move this to hoodie-common